README.md

```diff
@@ -106,15 +113,15 @@ spaceExport(options)
 })
 ```
 
-The `options` object can contain any of the CLI options, but written with a camelCase pattern instead, and no dashes. For example `--space-id` would become `spaceId`.
+The `opts` object within options can contain any of the CLI options, but written with a camelCase pattern instead and no dashes. For example `--space-id` would become `spaceId`.
 
 Another option is `errorLogFile`, the location of a file to log errors to.
 
 ## Limitations
 
-- This tool exports only your published Content Model, Content and Assets. It will support Editor Interfaces, Webhooks and Roles & Permissions in a future version. Draft entries are not supported.
-- You can only export the content exported by this tool into an empty space. We currently do not support merging content into pre-existing spaces.
-- If you have custom widgets, you need to reinstall them manually in the new space.
+- This tool currently does **not** support the export of roles & permissions and space memberships.
```

Comment:
@Khaledgarbaya can u clarify that roles & permissons (and webhook credentials) are the only things we do not support yet?